ghost-cat-app/src/app/View/app.module.ts

```diff
@@ -3,24 +3,30 @@ import { NgModule } from '@angular/core';
 import { MatButtonModule } from '@angular/material/button';
 import { MatDividerModule } from '@angular/material/divider';
 import { HttpClientModule } from '@angular/common/http';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MaterialModule } from './material.module';
 
```

Comment:
You might get a merge conflict here with Bennie's PR which makes the login page the home screen. The fix is going to be making sure that every import you added and every import Bennie added are all included in the final version of this file.

Reply:
Sorry! That comment ^ was me. I forgot I was logged in as the root account haha